query/math.go

```diff
@@ -30,6 +30,16 @@ type mathTree struct {
 	Child []*mathTree
 }
 
+var (
+	ErrorIntOverflow     = errors.New("Integer overflow")
+	ErrorFloatOverflow   = errors.New("Float overflow")
+	ErrorDivisionByZero  = errors.New("Division by zero")
+	ErrorModuloByZero    = errors.New("Modulo by zero")
```

Comment:
Combine these two. Python, for example, would return a `ZeroDivisionError` in both cases. Also, ensure that we only return this for integers.